kimjaegwan/src/components/CommentLayout.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { ChangeEvent, FormEvent, useRef, useState } from 'react';
+import styled from 'styled-components';
+import Comment from './Comment/Comment';
+import StyleSet from '../styles/styleSet';
+import { idText } from 'typescript';
+import Image from 'next/image';
+import submit from '../imgs/submit.png';
+interface IComment {
+    id: number;
+    name: string;
+    content: string;
+}
+const CommentLayout = () => {
+    // 에브리타임 클론코딩
+
+    const [comments, setComments] = useState<IComment[]>([
+        {
+            id: 0,
+            name: `익명1`,
+            content: '경주 여행 사진입니다! 에브리타임을 클론코딩 했습니다!',
+        },
+    ]);
+
+    const [currentComment, setCurrentComment] = useState<string>('');
+
+    const inputRef = useRef<HTMLInputElement>(null);
+
+    const _onCommentSubmit = (e: FormEvent<HTMLFormElement>) => {
+        e.preventDefault();
+        console.log(currentComment);
+
+        if (currentComment === '') {
+            alert('내용을 입력하세요.');
+            return;
+        }
+
+        setComments([
+            ...comments,
+            {
+                id: comments.length,
+                name: '익명' + (comments.length + 1),
+                content: currentComment,
+            },
+        ]);
+        setCurrentComment(() => '');
```

Comment:
```suggestion
        setCurrentComment('');
```
이런 경우라면 콜백 형태로 안쓰여도 될 거 같아요 !